packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/LegacyManifest.kt

```diff
@@ -107,14 +107,12 @@ class LegacyManifest private constructor(
         }
       }
       var runtimeVersion = rawManifest.getSDKVersion()
-      val runtimeVersionObject = rawManifest.getRuntimeVersion()
-      if (runtimeVersionObject != null) {
-        if (runtimeVersionObject is String) {
-          runtimeVersion = runtimeVersionObject
-        } else if (runtimeVersionObject is JSONObject) {
```

Comment:
I assume that this was never a dictionary in any production apps?

Reply:
Yes, confirmed this last week with @esamelson:

<img width="928" alt="Screen Shot 2021-06-15 at 5 12 07 AM" src="https://user-images.githubusercontent.com/1220444/122050332-3bfb9a80-cd98-11eb-9941-c89553a0ccaa.png">